docfx_yaml/extension.py

```diff
@@ -469,9 +469,13 @@ def _parse_docstring_summary(summary):
                 continue
 
         # Parse keywords if found.
-        if part.startswith(".."):
+        # lstrip is added to parse code blocks that are not formatted well.
+        # TODO: remove the lstrip if
+        # https://github.com/googleapis/gapic-generator-python/issues/1244 is
+        # resolved.
```

Comment:
Do we want to do this if older versions will always have the leading newlines?

Reply:
Nay, we should keep it for it to be backwards looking. Removed the to-do comment.